packages/expo-image/src/ExpoImage.web.tsx

```diff
@@ -150,6 +150,80 @@ const useTransition = (
   return { placeholder: {}, image: {} };
 };
 
+const findBestSourceForSize = (
+  sources: ImageSource[] | undefined,
+  size: DOMRect | null
+): ImageSource | null => {
+  return (
+    sources
+      // look for the smallest image that's still larger then a container
+      ?.map((source) => {
+        if (!size) return { source, penalty: 0, covers: false };
+        const { width, height } =
+          typeof source === 'object' ? source : { width: null, height: null };
+        if (width == null || height == null) return { source, penalty: 0, covers: false };
```

Comment:
```suggestion
        if (width == null || height == null) {
          return { source, penalty: 0, covers: false };
        }
```
I'm not a big fun of the if statement without brackets, but is up to you ;)

Reply:
Yeah, Tomek already mentioned that 😢